modules/core/src/main/java/org/apache/ignite/internal/util/lang/IgniteThrowableBiFunction.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.util.lang;
+
+import java.io.Serializable;
+import org.apache.ignite.IgniteCheckedException;
+
+/**
+ * Represents a bi function that accepts one argument and produces a result. Unlike {@link java.util.function.BiFunction}
+ * it is able to throw {@link IgniteCheckedException}.
+ *
+ * @param <T> The type of the first input to the function.
+ * @param <U> The type of the second input to the function.
+ * @param <R> The type of the result of the function.
+ */
+public interface IgniteThrowableBiFunction<T, U, R> extends Serializable {
```

Comment:
Is this you are looking for `ThrowableBiFunction`?

Reply:
Exactly, seems like you found a dead code.